packages/flame/lib/src/camera/viewfinder.dart

```diff
@@ -19,21 +18,17 @@ import 'package:meta/meta.dart';
 /// If you add children to the [Viewfinder] they will appear like HUDs i.e.
 /// statically in front of the world.
 class Viewfinder extends Component
-    with ParentIsA<CameraComponent>
     implements AnchorProvider, AngleProvider, PositionProvider, ScaleProvider {
-  /// Internal transform matrix used by the viewfinder.
-  final Transform2D _transform = Transform2D();
-
-  @internal
-  Transform2D get transform => _transform;
+  /// Transform matrix used by the viewfinder.
+  final Transform2D transform = Transform2D();
```

Comment:
I guess we can't prevent people from doing `viewport.transform.transformMatrix.storage[0] = 0` or something...

Reply:
Yeah, if they do that they'll have to suffer the consequences 😂